src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -1983,4 +1986,67 @@ public FileDragDropPreferenceType getEntryEditorFileLinkPreference() {
     public void storeEntryEditorFileLinkPreference(FileDragDropPreferenceType type) {
         put(ENTRY_EDITOR_DRAG_DROP_PREFERENCE_TYPE, type.name());
     }
+
+    @Override
+    public List<TemplateExporter> getCustomExportFormats(JournalAbbreviationLoader loader) {
+        int i = 0;
+        List<TemplateExporter> formats = new ArrayList<>();
+        String exporterName;
+        String filename;
+        String extension;
+        LayoutFormatterPreferences layoutPreferences = getLayoutFormatterPreferences(loader);
+        SavePreferences savePreferences = loadForExportFromPreferences();
+        List<String> formatData;
+        while (!((formatData = getStringList(CUSTOM_EXPORT_FORMAT + i)).isEmpty())) {
+            exporterName = formatData.get(EXPORTER_NAME_INDEX);
+            filename = formatData.get(EXPORTER_FILENAME_INDEX);
+            extension = formatData.get(EXPORTER_EXTENSION_INDEX);
+            Optional<TemplateExporter> format = Optional.of(new TemplateExporter(exporterName, filename, extension,
```

Comment:
This is somewhat strange. If you use `Optional.of(new ...)` then the optional is always non-empty. Thus, there is no need to use an optional to start with.